README.md

```diff
@@ -76,8 +76,28 @@ Feeding the data, url for download. Ultimately, the links have been sent to your
 
 ![](P2.gif?raw=true)
 
-### With Docker
-Run the following commands
+### With Docker (Use any of the two methods)
+#### Pull Docker image from Docker Hub and start the container
+Make sure you are logged into Docker Hub from your command line by running `docker login`
+Now run the following commands
+```
+$ docker pull ashwani99/moboff
+
+# disable access control for X11 server
+$ xhost +
+
+# run the container
+$ docker run -it -e DISPLAY=$DISPLAY \
```

Comment:
Assuming a user repeats the steps every time to convert, we should either add `-rm` so that, once the user has exited the container, the container gets killed or add `-name` to the container and add instructions to using it next time, which would be `docker start -a <name>`